java/src/jmri/jmrix/zimo/Mx1SystemConnectionMemo.java

```diff
@@ -141,6 +141,11 @@ public boolean provides(Class<?> type) {
      * hexfile.HexFileFrame and locormi.LnMessageClient
      */
     public void configureManagers() {
+        
+        /* Already created during config with default settings. Recreating
+        * now that the traffic controller exists
+        */
+        programmerManager = null;
```

Comment:
An alternative approach could possibly not create the default programmer at all and defer creation until this point,  This seemed safer as I did not know the consequences for the former approach. Other better alternatives could also exist?

This fix restores CV read / write when using decoder pro.

Reply:
I think it would be better to defer creation until after the traffic controller is created.

It is possible the earlier creation is technical debt from a previous incarnation.

Reply:
Possibly, I'm not super familiar with the architecture.  Let me try to understand if we could defer creation easily or not.  Is using the configure managers method the preferred approach?  I'll try to get all creation to occur within this method.